ProcessMaker/Http/Controllers/Api/ProcessController.php

```diff
@@ -370,6 +370,9 @@ public function update(Request $request, Process $process)
             );
         }
 
+        //Call Event to Log Template Changes
+        TemplateUpdated::dispatch([], $process, 'Update Template Process');
```

Comment:
use a const instead of a hardcoded value 'Update Template Process' like:
`TemplateUpdated::UPDATE_TEMPLATE`